tests/metactl/test-metactl.sh

```diff
@@ -48,3 +50,16 @@ echo " === check if there is a node record in it"
 grep -Fxq '["raft_state",{"RaftStateKV":{"key":"Id","value":{"NodeId":0}}}]' $grpc_exported
 
 kill $METASRV_PID
+
+echo " === export data from a running databend-meta to $grpc_exported with initial-cluster args"
+./target/${BUILD_PROFILE}/databend-metactl --import --raft-dir $import_meta_dir --db $grpc_exported --initial-cluster 3=http://localhost:23822,0.0.0.0:9292
```

Comment:
I think we should test on whether we could retrieve the same result from new cluster recovered from metactl

Reply:
add test-metactl-restore-new-cluster.sh to test this case.